cfgov/v1/atomic_elements/organisms.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa F401
```

Comment:
Would it make sense to just do this as a `# noqa: F401` on the line? I'm a little wary of ignore unused imports in the whole file. Not a blocker, though.